hail/python/hail/linalg/blockmatrix.py

```diff
@@ -579,6 +579,16 @@ def write(self, path, overwrite=False, force_row_major=False, stage_locally=Fals
         writer = BlockMatrixNativeWriter(path, overwrite, force_row_major, stage_locally)
         Env.backend().execute(BlockMatrixWrite(self._bmir, writer))
 
+    @typecheck_method(path=str,
+                      overwrite=bool,
+                      force_row_major=bool,
+                      stage_locally=bool)
+    def checkpoint(self, path, overwrite=False, force_row_major=False, stage_locally=False):
+        """Checkpoint the block matrix. See :func:`.write` for explanation of arguments.
```

Comment:
If we're not going to document the arguments, I think it's better to use `(path, *args, **kwargs)` and call write as `write(path, *args, **kwargs)`. That ensures the functions always have the same signature.

Reply:
we should document the arguments here, it's user-exposed